base/math.jl

```diff
@@ -851,26 +851,59 @@ minmax(x::T, y::T) where {T<:AbstractFloat} = min(x, y), max(x, y)
 
 _isless(x::Float16, y::Float16) = signbit(widen(x) - widen(y))
 
+const _minmaxtypes = [(:f64, :Float64, :double), (:f32, :Float32, :float)] #TODO: Add a way to detect native f16
```

Comment:
If you put into a let block you don't need to keep the globals, around, right?

Reply:
Thinking about it, probably the ccall with llvmcall syntax is a lot cleaner